### DISCUSSION TITLE
Problem in displaying transition using _hover

### DISCUSSION THREAD

**Discussion by VikaramjitSinghGorraya** [chakra-ui/chakra-ui#3094] [Q&A]
I tried calling onToggle inside _hover, but it shows an error saying that maximum depth has been reacherd. Below is the snippet of the code:-
 const { isOpen, onToggle } = useDisclosure()

 <Fade in={isOpen}>
                Fade
            </Fade>
 <Box 
                bgImage = {`url(https://static01.nyt.com/${article.multimedia[0].url})`} 
                bgSize = "cover"
                position = 'relative'
                h = '200px' 
                w = '100%'
                _hover = {onToggle}
                />

**Comment by chasinhues** [comment]
@VikaramjitSinghGorraya hey there 👋  

Can you please provide some more info on this question? What were you trying to accomplish by calling `onToggle` inside the `_hover` pseudo prop?

Showing more of your code would be handy too, that single line doesn't really give us anything to help you with 😅
